package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@criticalmanufacturing/dev-tasks",
-  "version": "8.1.0",
+  "version": "8.1.0-0",
```

Comment:
8.1.0 is already out. Please change it to 8.2.0-0

Reply:
done.